Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:0.10.36
+FROM node:7.3
```

Comment:
So I changed `lru_map` to `lru-cache` and all seem to work now without touching the base node image. I guess this is the best solution.